docs/details.md

```diff
@@ -5,7 +5,8 @@
 * The target device for this contest will be the AMD/Xilinx UltraScale+ xcvu3p.
 * Competing routers must consume a pre-placed and partially-routed
  [FPGA Interchange Format](http://www.rapidwright.io/docs/FPGA_Interchange_Format.html) Physical Netlist
-  and emit a fully routed Physical Netlist.
+  and emit a fully routed Physical Netlist that is identical to the input, except for the addition of
+  routeSegments of the type `pip`.
```

Comment:
```suggestion
  and emit a fully routed Physical Netlist formed by enabling some number of routing switches (termed PIPs).
```

I would be more specific (that routeSegment `pip` objects are the only thing that can be inserted into the netlist) in the Router section below.